Class/Practise 10.1/src/Team.cpp

```diff
@@ -0,0 +1,28 @@
+#include "Team.hpp"
+
+Team::Team(const std::string& name) 
+    : name(name), points(0), totalTopHeight(0), totalGoalsScored(0), totalGoalsConceded(0), yellowCards(0), redCards(0) {}
```

Comment:
there is no need to mention all the fields. Just add that default values to field declaration